python/oneflow/test/modules/test_free_tensor_not_in_job.py

```diff
@@ -0,0 +1,169 @@
+"""
+import unittest
+import numpy as np
+
+import oneflow as flow
+import oneflow.unittest
+import oneflow.nn as nn
+
+
+class OneEmbedding(nn.Module):
+    def __init__(
+        self,
+        table_name,
+        embedding_vec_size,
+        persistent_path,
+        table_size_array,
+        store_type,
+        cache_memory_budget_mb,
+        size_factor,
+    ):
+        assert table_size_array is not None
+        vocab_size = sum(table_size_array)
+
+        tables = [
+            flow.one_embedding.make_table(
+                flow.one_embedding.make_normal_initializer(mean=0.0, std=1)
+            )
+            for _ in range(len(table_size_array))
+        ]
+        if store_type == "device_mem":
+            store_options = flow.one_embedding.make_device_mem_store_options(
+                persistent_path=persistent_path,
+                capacity=vocab_size,
+                size_factor=size_factor,
+            )
+        elif store_type == "cached_host_mem":
+            assert cache_memory_budget_mb > 0
+            store_options = flow.one_embedding.make_cached_host_mem_store_options(
+                cache_budget_mb=cache_memory_budget_mb,
+                persistent_path=persistent_path,
+                capacity=vocab_size,
+                size_factor=size_factor,
+            )
+        elif store_type == "cached_ssd":
+            assert cache_memory_budget_mb > 0
+            store_options = flow.one_embedding.make_cached_ssd_store_options(
+                cache_budget_mb=cache_memory_budget_mb,
+                persistent_path=persistent_path,
+                capacity=vocab_size,
+                size_factor=size_factor,
+            )
+        else:
+            raise NotImplementedError("not support", store_type)
+
+        super(OneEmbedding, self).__init__()
+        self.one_embedding = flow.one_embedding.MultiTableEmbedding(
+            name=table_name,
+            embedding_dim=embedding_vec_size,
+            dtype=flow.float,
+            key_type=flow.int64,
+            tables=tables,
+            store_options=store_options,
+        )
+
+    def forward(self, ids):
+        return self.one_embedding.forward(ids)
+
+
+class my_model(nn.Module):
+    def __init__(
+        self,
+        embedding_vec_size,
+        persistent_path,
+        table_size_array,
+        cache_memory_budget_mb,
+        one_embedding_store_type,
+        size_factor,
+    ):
+        super(my_model, self).__init__()
+
+        self.embedding_layer = OneEmbedding(
+            table_name="sparse_embedding",
+            embedding_vec_size=embedding_vec_size,
+            persistent_path=persistent_path,
+            table_size_array=table_size_array,
+            store_type=one_embedding_store_type,
+            cache_memory_budget_mb=cache_memory_budget_mb,
+            size_factor=size_factor,
+        )
+
+        self.linear_1 = nn.Linear(6, 6)
+        self.batch_norm = nn.BatchNorm1d(6)
```

Comment:
This test should be placed in `python/oneflow/test/graph`.

A simple nn.BatchNorm1d can test this fix, it's better to simple the test?